File: Import/Managers/IObjectManager.cs
using System.Data;

namespace kCura.WinEDDS.Core.Import.Managers
{
	public interface IObjectManager
	{
		DataSet RetrieveArtifactIdOfMappedParentObject(int caseContextArtifactId, string textIdentifier, int artifactTypeId);
	}
}

File: Source/Relativity.DataExchange.Legacy/Service/Replacement/IObjectManager.vb
Namespace kCura.WinEDDS.Service.Replacement
    Public interface IObjectManager
        Inherits IDisposable
        Shadows Function RetrieveArtifactIdOfMappedObject(ByVal caseContextArtifactID As Int32, ByVal textIdentifier As String, ByVal artifactTypeID As Int32) As System.Data.DataSet
        Shadows Function RetrieveTextIdentifierOfMappedObject(ByVal caseContextArtifactID As Int32, ByVal artifactId As Int32, ByVal artifactTypeID As Int32) As System.Data.DataSet
        Shadows Function RetrieveArtifactIdOfMappedParentObject(ByVal caseContextArtifactID As Int32, ByVal textIdentifier As String, ByVal artifactTypeID As Int32) As System.Data.DataSet
    end interface
End NameSpace